index.js

```diff
@@ -13,28 +16,60 @@ module.exports = {
       name: 'ember-cli-babel',
       ext: 'js',
       toTree: function(tree) {
-        return require('broccoli-babel-transpiler')(tree, getOptions(addon));
+        return require('broccoli-babel-transpiler')(tree, getBabelOptions(addon));
       }
     };
 
     registry.add('js', plugin);
   },
 
+  shouldIncludePolyfill: function() {
+    var options = getAddonOptions(this);
+    return options.includePolyfill === true;
+  },
+
+  importPolyfill: function(app) {
+    app.import('vendor/browser-polyfill.js');
```

Comment:
Probably wouldn't hurt, but is it likely to make an observable difference? Obviously Ember can't rely on ES6 APIs being available, but I'm not sure whether it behaves any differently in their presence.